diesel_migrations/Cargo.toml

```diff
@@ -9,11 +9,11 @@ homepage = "https://diesel.rs"
 
 
 [dependencies]
-migrations_internals = "~1.4.0"
-migrations_macros = "~1.4.0"
+migrations_internals = "=1.4.0"
```

Comment:
It is probably worth to cherry-pick [this](https://github.com/diesel-rs/diesel/commit/7176fe98af2a57dc4365a7682eb2f9b7dce72121#diff-d488e04098f2e4da9d4b763714a9e653) commit as well, as it should fix this kind of issues.

Reply:
This looks like it could work quite well. In fact it is a good idea and should be kept in mind. On the same line of thought `diesel-oci` should be moved into the Giga-infosystems branch of `diesel` so that we are always using a compatible commit of `diesel-oci`.

However, I do think that it is not directly relevant (okay, it is, but it isn't) to the exact task at hand and should be done later as a "general improvement" for when we next bump the versions.

Reply:
(That is to say that I will definitely merge have a play with commit, but I would like to have all the "essentials" for the logger (and the logger) merged first, and then apply the "nice-to-haves").

Reply:
In fact when this is added then we can also bump the version to `1.4.5`, I think it would be quite representative of the change.